.github/workflows/artifacts.yaml

```diff
@@ -75,6 +75,9 @@ jobs:
         path: ${{ env.x86_64_image_path }}
   export-docker-image-arm64:
```

Comment:
We have another job `export-docker-image-x86_64` above. If it does for arm64, why don't we do it for x86_64 as well?

Reply:
x86_64 is run on Github-hosted runner, so it would be unnecessary